### PR TITLE
Implement Mock DB and Express Server

### PR DESCRIPTION
**Pull request by miicow**
1. Querying DB for a single data entry.
2. Code clean up.

### CODE REVIEW

File: database/seed.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    name: `${capitalize(faker.hacker.adjective())} ${capitalize(
+      faker.hacker.noun(),
+    )}`,
+    imageUrl: faker.image.imageUrl(),
+    createdAt: faker.date.past(),
+  });
+}
+
+function insertSampleRest() {
+  Restaurant.Restaurant.create(sampleRestaurant);
```
Comment (tlindow) [inline database/seed.js:22]: Maybe es6 destructuring could help here when you required the module? This syntax is feels redundant with 'Restaurant.Restaurant'.

File: server/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+const Restaurant = require('../database/Restaurant.js');
+
+const app = express();
+
+app.use(morgan('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.get('/id/:id', (req, res) => {
+  Restaurant.findById('5c9e9d340b52d24f643c655a', (err, result) => {
```
Comment (tlindow) [inline server/index.js:14]: Can you edit the id in this line to reflect a more general id? Let's move this specific id to one your tests

File: database/seed.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+const Restaurant = require('./Restaurant.js');
+
+const sampleRestaurant = [];
+
+const capitalize = (s) => {
+  if (typeof s !== 'string') return '';
+  return s.charAt(0).toUpperCase() + s.slice(1);
+};
+
+for (let i = 1; i <= 100; i += 1) {
```
Comment (tlindow) [inline database/seed.js:11]: The data generation section is very clear! Nice!

**Review by tlindow — APPROVED** [review]
Looks great overall! The most important fix in a future commit will be to edit server/index.js to take in a more general id and leave the specifics to a test file. I'll merge this PR! -Tyler

File: server/index.js (hunk 2 above)
Comment (miicow) [reply server/index.js:14]: The specific ID was used as initially to see that I was able to query the DB, will change as I implement more and more.

Comment (dorktobes) [reply server/index.js:14]: I'd also argue that the endpoint `/id` is not descriptive of what data would be returned to a query here. the noun should describe the resource.

**Pull request merged**